disclosure/urls.py

```diff
@@ -16,9 +16,7 @@
                                'document_root': settings.STATIC_ROOT,
                                'show_indexes': True,
                            }),
-                       url(r'', include('locality.urls')),
-                       url(r'^locations/(?P<fips_id>[0-9]+)$',
-                           views.location_view),
```

Comment:
@adborden Is there a front-end readout of this API endpoint? And is there any list of data stubbed in the front-end code? 

Connecting those up will help understand whether the data this PR lives on top of looks good enough for front-end devs to work with. The "search" one is fine, but any others would really help!